<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BotControlGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="Serial_groupBox">
        <property name="title">
         <string>Serielle Verbindung</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="HLayOutSerialSettings">
           <item>
            <widget class="QComboBox" name="portsComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refreshPortsPushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>38</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>38</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="mainWindow.qrc">
               <normaloff>:/icons/icons/refresh.png</normaloff>:/icons/icons/refresh.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baudRateComboBox">
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="connectPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="disconnectPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="logPlainTextEdit">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cmdLineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>&gt;LedBlinkTime=100&lt;</string>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="PID_groupBox_2">
          <property name="title">
           <string>PID Regler</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="D_Label">
             <property name="text">
              <string>D :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="I_Label">
             <property name="text">
              <string>I :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="P_ValueLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="I_ValueLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSlider" name="P_HSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>3000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="pageStep">
              <number>300</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="D_ValueLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="P_Label">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>P :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSlider" name="D_HSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>3000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="pageStep">
              <number>300</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="I_HSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>3000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="pageStep">
              <number>300</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="FreeSlider_groupBox_3">
          <property name="title">
           <string>Freie Regler</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="FreeSlider2_Label">
             <property name="text">
              <string>Max Geschwindigkeit:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="FreeSlider1_Label">
             <property name="text">
              <string>Max Beschleunigung :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="FreeSlider2_HSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>3000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="pageStep">
              <number>300</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="FreeSlider2_ValueLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSlider" name="FreeSlider1_HSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>3000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="pageStep">
              <number>300</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="FreeSlider1_ValueLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="FreeSwitches_groupBox">
          <property name="title">
           <string>Freie Schalter</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="Free_Free_checkBox_1">
             <property name="text">
              <string>Schalter 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="Free_checkBox_2">
             <property name="text">
              <string>Schalter 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="Free_checkBox_3">
             <property name="text">
              <string>Schalter 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="Free_checkBox_4">
             <property name="text">
              <string>Schalter 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="Free_checkBox_5">
             <property name="text">
              <string>Schalter 5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MplWidget" name="mpl_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>logPlainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>P_HSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>P_ValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>I_HSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>I_ValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>D_HSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>D_ValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FreeSlider1_HSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FreeSlider1_ValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FreeSlider2_HSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FreeSlider2_ValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
